.gitignore

```diff
@@ -66,3 +66,5 @@ db.sqlite3
 /data/
 src/backend/joanie/static/joanie/
 
+# # IDEs
+.idea/*
```

Comment:
I think you could add also `*.iml` files
And also `.vscode` folder

Reply:
ok I add them